### PR TITLE
CORE-9360: Fix Mgm plugin

### PR DESCRIPTION
**Pull request by yift-r3**
* Add temporary support for CPI version 2.
* Set `offsetsTopicReplicationFactor` and `transactionStateLogReplicationFactor` to 1 (so it will work with any cluster size).

### CODE REVIEW

**Review by josephzunigadaly — DISMISSED** [review]

File: tools/plugins/mgm/src/main/kotlin/net/corda/cli/plugins/mgm/OnBoardMember.kt [hunk 1]
```diff
@@ -34,8 +31,8 @@ class OnBoardMember : Runnable, BaseOnboard() {
     @Option(
         names = ["--cpb-file"],
         description = [
-            "Location of a CPI file (Use either cpi-file, cpb-file or cpi-hash).",
-            "To create the CPI use the MGM created group policy file and sign it using the package create-cpi command."
+            "Location of a CPB file (Use either cpi-file, cpb-file or cpi-hash).",
+            "Usage of --cpi-file or --cpi-hash is recommended. The CPI wil be sign with default options."
```
Comment (charlieR3) [inline tools/plugins/mgm/src/main/kotlin/net/corda/cli/plugins/mgm/OnBoardMember.kt]:
```suggestion
            "Usage of --cpi-file or --cpi-hash is recommended. The CPI will be signed with default options."
```

**Review by charlieR3 — COMMENTED** [review]
One comment on wording but otherwise looks good

**Review by davidcurrie — COMMENTED** [review]
I don't know where this plugin is used but `offsetsTopicReplicationFactor` and `transactionStateLogReplicationFactor` were set to 3 in the E2E tests so that they could withstand the failure of Kafka brokers whilst the tests were running. If the plugin is ever used to deploy supposedly HA clusters then these should probably only be set to 1 if the number of replicas is less than the replication factor.

**Review by davidcurrie — APPROVED** [review]
👍 for the helm part

**Review by ronanbrowne — APPROVED** [review]
Lgtm - build

**Pull request merged**